src/runtime/d3d12compute.cpp

```diff
@@ -1897,8 +2076,32 @@ static void *buffer_contents(d3d12_buffer *buffer) {
             pData = map_buffer(&readback);
             break;
 
-        case d3d12_buffer::Unknown :
+        case d3d12_buffer::ReadOnly  :
```

Comment:
aligning the colons is weird and I don't think we do it elsewhere. Just make them flush.